x-pack/plugins/security_solution/common/search_strategy/security_solution/users/authentications/index.ts

```diff
@@ -23,7 +23,7 @@ import type { CommonFields, RequestOptionsPaginated } from '../..';
 
 export interface UserAuthenticationsStrategyResponse extends IEsSearchResponse {
   edges: AuthenticationsEdges[];
-  totalCount: number;
+  totalCount: number | null | undefined;
```

Comment:
per PR description, default totalCount is changed to `undefined` from `-1`. In which cases totalCount can be `null`?